src/features/bookings/helpers.test.ts

```diff
@@ -605,3 +606,56 @@ describe('getOfferRules', () => {
     expect(offerRules).toEqual('')
   })
 })
+
+describe('getLocationLabel', () => {
+  const initialBooking = bookingsSnap.ongoing_bookings[1]
+
+  it('should return empty string if permanent offer', () => {
+    const properties = {
+      isPermanent: true,
+    }
+    const offerRules = getLocationLabel(initialBooking.stock, properties)
+    expect(offerRules).toEqual('')
+  })
+
+  it('should return empty string if digital offer', () => {
+    const properties = {
+      isDigital: true,
+    }
+    const offerRules = getLocationLabel(initialBooking.stock, properties)
+    expect(offerRules).toEqual('')
+  })
+
+  it('should display the name of the venue if public name not informed', () => {
```

Comment:
que penserais-tu de ?

```suggestion
  it('should display the name of the venue when public name is not set', () => {
```